frappe/utils/safe_exec.py

```diff
@@ -445,8 +445,8 @@ def add_module_properties(module, data, filter_method):
 	"now_datetime",
 	"get_timestamp",
 	"get_eta",
-	"get_time_zone",
-	"convert_utc_to_user_timezone",
```

Comment:
this might break all the server scripts - let's not merge the other pr for now

or we can even keep the aliases ...(?)

Reply:
Right, this should go into the other PR